commands/management/deregister.go

```diff
@@ -0,0 +1,72 @@
+// pmm-admin
+// Copyright 2021 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package management
+
+import (
+	"github.com/percona/pmm-admin/agentlocal"
+	"github.com/percona/pmm-admin/commands"
+	"github.com/percona/pmm/api/inventorypb/json/client"
+	"github.com/percona/pmm/api/inventorypb/json/client/nodes"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+type deregisterCommand struct {
```

Comment:
IMO one tiny change  required imho: `unregister` vs `deregister`. Full-blown explanation and reasons given [on this resource](https://english.stackexchange.com/questions/25931/unregister-vs-deregister)
Sorry for the trouble :(